### ISSUE TITLE
Enable environment-file: with tests, as another way to set tmt variables.

### ISSUE THREAD

**Issue by sbertramrh** [teemtee/tmt#1255]
Currently environment-file: option is only available to plans and there are instances where this would also be useful and helpful with tests. Request is to extend this functionality to tests as well.

https://tmt.readthedocs.io/en/stable/spec/plans.html#environment-file

A note, if the decision is not to extend the functionality, then an error should be thrown when used in a test. Currently no errors are thrown during execution so user may assume that this is currently a valid option for tests.

**Comment by psss** [comment]
> Currently no errors are thrown during execution so user may assume that this is currently a valid option for tests.

Yes, we are currently working on json schemas which will be used for checking correct config syntax, so this should be covered soon as well.
